<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculateShipLocation</class>
 <widget class="QDialog" name="CalculateShipLocation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>423</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(240, 240, 240);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Current User</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leCurrentUser">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>133</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_10">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Users </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="usersList">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="currentText">
         <string/>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QFrame" name="calculate">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(233, 233, 235);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Ship ID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Ship Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLineEdit" name="leName">
        <property name="maximumSize">
         <size>
          <width>133</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5" rowspan="6">
       <widget class="QWidget" name="dialWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Latitude</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="latitude">
        <property name="maximumSize">
         <size>
          <width>133</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>55</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Longitude</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLineEdit" name="longitude">
        <property name="maximumSize">
         <size>
          <width>133</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Speed</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="speed">
        <property name="maximumSize">
         <size>
          <width>133</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>10</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Angle</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLineEdit" name="angle">
        <property name="maximumSize">
         <size>
          <width>133</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Set Time</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="setTime">
        <property name="maximumSize">
         <size>
          <width>133</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Receiver</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QLineEdit" name="receiver">
        <property name="maximumSize">
         <size>
          <width>133</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>KEY_NODE</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="calculate_2">
        <property name="text">
         <string>Calculate</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3" colspan="2">
       <widget class="QPushButton" name="sendPosition">
        <property name="text">
         <string>Start Sending Position</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lbShipId">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>label_2</zorder>
     <zorder>label_5</zorder>
     <zorder>longitude</zorder>
     <zorder>latitude</zorder>
     <zorder>label_3</zorder>
     <zorder>angle</zorder>
     <zorder>label_4</zorder>
     <zorder>speed</zorder>
     <zorder>calculate_2</zorder>
     <zorder>sendPosition</zorder>
     <zorder>label_6</zorder>
     <zorder>setTime</zorder>
     <zorder>label_7</zorder>
     <zorder>receiver</zorder>
     <zorder>label_8</zorder>
     <zorder>lbShipId</zorder>
     <zorder>dialWidget</zorder>
     <zorder>label</zorder>
     <zorder>leName</zorder>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_9">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Sent Messages</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QTextEdit" name="displayLocation">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="pbClear">
     <property name="minimumSize">
      <size>
       <width>125</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="stopSending">
     <property name="minimumSize">
      <size>
       <width>141</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Stop Sending</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbClear</sender>
   <signal>clicked()</signal>
   <receiver>displayLocation</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>653</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>555</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
